index.js

```diff
@@ -1,6 +1,10 @@
+import 'react-dates/initialize';
```

Comment:
We should require the integrating library to do this in case react-dates is used in other places as well.

It's quite a common date picker for us an we use it as the standard date picker (not just in tables) so we should account for not initializing it twice. I recommend we leave the initialization to the parent app